### DISCUSSION TITLE
[PL] Przypisanie wartości z produktu do nowych pól w bloku zaawansowanym

### DISCUSSION THREAD

**Discussion by hahyes** [YetiForceCompany/YetiForceCRM#14131] [How to] [answered]
Witam,

chciałem za pomocą konfiguracji modułów dodać do bloku zaawansowanego nowe pola w wierszu podstawowym i podpiąć pod nie nowe pola z produktu. 

Co trzeba zrobić, żeby wartości przypisywały się prawidłowo? Jakiś odpowiedni typ pola, który muszę wybrać? A może nazwa etykiety musi się zgadzać?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/issues/12945
